SBR.md

```diff
@@ -2360,6 +2364,10 @@ The Repository MAY include CRL entries that have a `CRLreason` of certificateHol
 
 If a `reasonCode` CRL entry extension is present, the `CRLReason` SHALL indicate the most appropriate reason for revocation of the Certificate, as defined by the CA within its CP/CPS.
 
+### 7.2.3 CRL entry revocationDate field
+
+The CA SHOULD update the revocation date in a CRL entry when it is determined that the Private Key of the Certificate was compromised prior to the revocation date that is indicated in the CRL entry for that Certificate. Backdating the revocationDate field is an exception to best practice described in RFC 5280 (Section 5.3.2); however, these Requirements specify the use of the revocationDate field to support S/MIME implementations that process the revocationDate field as the date when the Certificate is first considered to be compromised.
```

Comment:
I think we need a bit of clarity on whether or not it's acceptable to backdate revocations for reasons other than Key Compromise. For example, if it is discovered that a Subscriber violated their obligations under the Subscriber Agreement several months ago, is it appropriate or allowed for the CA to set the `revocationDate` several months in the past?

Reply:
> I think we need a bit of clarity on whether or not it's acceptable to backdate revocations for reasons other than Key Compromise. For example, if it is discovered that a Subscriber violated their obligations under the Subscriber Agreement several months ago, is it appropriate or allowed for the CA to set the `revocationDate` several months in the past?

For example... It was discovered that the identity validation was not done properly, could it be appropriate to backdate to the same issuance time, so any digital signature is deemed invalid for that certificate?

BTW, could it be that certificate consumers aren't currently checking the revocation date when opening an email, but just that the certificate is revoked?

Reply:
@CBonnell  and @pfuentes69 both good points, I had not even considered that scenario, thank you. 

Lets discuss tomorrow during the S/MIME call

Reply:
In my opinion, alignment with RFC 5280 is better.  This might get S/MIME UAs to process the CRL entry to give better information to the user.  I would prefer to say:

~~~
### Invalidity Date CRL Entry Extension

The CA SHOULD provide the Invalidity Date CRL entry extension when it is determined that the Private Key associated with the Certificate was compromised prior to the revocation date that is indicated in the CRL entry for that Certificate. 
~~~